feature-flags.json

```diff
@@ -1 +1 @@
-[ "gm3Components" ]
```

Comment:
This is not obvious at the moment due to there being only one existing feature flag, but we prefer keeping the feature flags in alphabetical order. Can you update this? Cheers